files/k3s-install-agent.sh

```diff
@@ -12,32 +12,76 @@ do
 done
 }
 
-# Disable firewall 
-/usr/sbin/netfilter-persistent stop
-/usr/sbin/netfilter-persistent flush
-
-systemctl stop netfilter-persistent.service
-systemctl disable netfilter-persistent.service
-
-# END Disable firewall
+if test -f /etc/lsb-release; then
+  operating_system="ubuntu"
+else
+  operating_system="oraclelinux"
+fi
```

Comment:
I suspect a more reliable test would be to `source /etc/os-release` and then use the value of `$NAME` and `$VERSION_ID` or perhaps even `$PRETTY_NAME`. This would work regardless of whether LSB compatibility packages are installed or not and it's pretty much what `/etc/os-release` was designed for.

Reply:
Fixed in commit 9eac863